src/components/SettingsPluginsBlock.vue

```diff
@@ -14,6 +14,9 @@ const modalPluginsOpen = ref(false);
 
 function handleEditPlugins(name: string) {
   if (props.isViewOnly) return;
+  // the oSnap plugin does not require any configuration
+  // so we don't need to open the modal
+  if (name === 'oSnap') return;
```

Comment:
I think we should handle it like we do when selecting the plugin in the modal, when the plugin doesn't have `defaults` defined

https://github.com/snapshot-labs/snapshot/blob/335f6f2d59546023c96c17c4a763c8765fce02a9/src/components/ModalPlugins.vue#L32

Reply:
could you go into more detail? i dont understand if you want me to remove the changes in the `handleEditPlugins` and instetad put it in the `selectPlugin` function instead, or move the default checking from `selectPlugin` to `handleEditPlugins` function